src/test/java/hudson/maven/MavenJenkinsRule.java

```diff
@@ -5,8 +5,10 @@
 import org.jvnet.hudson.test.JenkinsRule;
 
 import java.io.File;
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URISyntaxException;
+import java.net.URL;
 
 /**
  * Customisation of the JenkinsRule to fix the lost of window focus when tests are launched on MacOS.
```

Comment:
What exactly is that problem and why is it not being fixed in `jenkins-test-harness`?

Reply:
Dunno, prefer not to touch fo now